docs/source/examples/dataframe-hdf5.rst

```diff
@@ -0,0 +1,152 @@
+Dataframes from HDF5 files
+=========================
+
+This section provides working examples of ``dask.dataframe`` methods to read HDF5 files. HDF5 is a unique technology suite that makes possible the management of extremely large and complex data collections. To learn more about HDF5, visit the `HDF Group Tutorial page <https://www.hdfgroup.org/HDF5/whatishdf5.html>`_.  For an overview of ``dask.dataframe``, its limitations, scope, and use, see the `DataFrame Overview section <http://dask.pydata.org/en/latest/dataframe-overview.html>`_.  
+
+  **Important Note** -- `h5py <http://docs.h5py.org/en/latest/index.html>`_ and `PyTables <http://www.pytables.org/index.html>`_ are the two premiere libraries to work with HDF5 data in Python.   ``dask.dataframe``, `like python pandas <http://pandas.pydata.org/pandas-docs/stable/io.html#io-hdf5>`_, uses the the `PyTables <http://www.pytables.org/FAQ.html#what-is-pytables>`_ library to interface with the HDF5 binary data format. HDF5 files made using the h5py project may not read into a dask DataFrame. 
```

Comment:
Great point.  Changed the entire "note" section to these two lines:

**Important Note** -- ``dask.dataframe.read_hdf`` uses ``pandas.read_hdf``, thereby inheriting its abilities and limitations.  See <http://pandas.pydata.org/pandas-docs/stable/io.html#hdf5-pytables>`_ for more information.